src/script/page/MainContent/panels/preferences/accountPreferences/AccountSecuritySection.tsx

```diff
@@ -33,7 +33,7 @@ import {useKoSubscribableChildren} from 'Util/ComponentUtil';
 import {t} from 'Util/LocalizerUtil';
 import {safeWindowOpen} from 'Util/SanitizationUtil';
 import PreferencesSection from '../components/PreferencesSection';
-import PreferencesLink from '../components/PreferencesLink';
+import {Link, LinkVariant} from '@wireapp/react-ui-kit';
```

Comment:
can you check if `PreferencesLink` is still used, it seems not.
In which case we can remove the file altogether

Reply:
Noted :+1:  I'll do a last clean-up of the preferences soon